src/topp/SageAdapter.cpp

```diff
@@ -420,6 +1057,16 @@ class TOPPSageAdapter :
     registerStringList_("variable_modifications", "<mods>", ListUtils::create<String>("Oxidation (M)", ','), "Variable modifications, specified using Unimod (www.unimod.org) terms, e.g. 'Carbamidomethyl (C)' or 'Oxidation (M)'", false);
     setValidStrings_("variable_modifications", all_mods);
 
+    //FDR and misc 
+    registerDoubleOption_("FDR_Threshhold", "<double>", 0.01, "The FDR threshhold for filtering peptides", false, false); 
+    registerStringOption_("Annotate_matches", "<bool>", "false", "If the matches should be annotated (default: false),", false, false); 
```

Comment:
maybe don't mix capitalization styles here. e.g. annotate_matches would fit better to predict_rt option

Reply:
also above "max_q_value" or "q_value_threshold"

Reply:
Thanks, I've changed the option names now.